certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+        "interest": {
+          "format": "double",
+          "description": "Gets or Sets Interest",
+          "type": "number"
+        },
+        "mortgageInsurance": {
+          "format": "double",
+          "description": "Gets or Sets MortgageInsurance",
+          "type": "number"
+        },
+        "taxes": {
+          "format": "double",
+          "description": "Gets or Sets Taxes",
+          "type": "number"
+        },
+        "hoa": {
+          "format": "double",
+          "description": "Gets or Sets Hoa",
+          "type": "number"
+        },
+        "totalMonthly": {
+          "format": "double",
+          "description": "Gets or Sets TotalMonthly",
+          "type": "number"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.LoanDetail": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "rate": {
+          "format": "double",
+          "description": "Gets or Sets Rate",
+          "type": "number"
+        },
+        "apr": {
+          "format": "double",
+          "description": "Gets or Sets Apr",
+          "type": "number"
+        },
+        "term": {
+          "format": "int32",
+          "description": "Gets or Sets Term",
+          "type": "integer"
+        },
+        "payment": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.LoanPayment"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Loan1": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sequence": {
+          "format": "int32",
+          "description": "Gets or Sets Sequence",
+          "type": "integer"
+        },
+        "id": {
+          "format": "int32",
+          "description": "Gets or Sets Id",
+          "type": "integer"
+        },
+        "loanNumber": {
+          "description": "Gets or Sets LoanNumber",
+          "type": "string"
+        },
+        "amount": {
+          "format": "double",
+          "description": "Gets or Sets Amount",
+          "type": "number"
+        },
+        "loanPolicyLiability": {
+          "format": "double",
+          "description": "Gets or Sets LoanPolicyLiability",
+          "type": "number"
+        },
+        "lender": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "mortgageBroker": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.PartyReference"
+        },
+        "mortgageInsuranceCase": {
+          "description": "Gets or Sets MortgageInsuranceCase",
+          "type": "string"
+        },
+        "loanDetail": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.LoanDetail"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Financial": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.